docs/import/manage_layers.js

```diff
@@ -328,31 +327,10 @@ function populateStateAndDisasterAssetPickers(disaster) {
   for (const state of states) {
     if (!stateAssets.has(state)) statesToFetch.push(state);
   }
-  // TODO: add functionality to re-pull all cached states from ee without
-  // reloading the page.
-  if (statesToFetch.length === 0) {
-    createStateAssetPickers(states);
-  } else {
-    const statesDone = getStatesAssetsFromEe(statesToFetch).then((assets) => {
-      for (const asset of assets) {
-        stateAssets.set(asset[0], asset[1]);
-      }
-      createStateAssetPickers(states);
-    });
-    promises.push(statesDone);
-  }
 
```

Comment:
I also think you can get rid of the stateAssets map for this page entirely (but would double check that!)